game/tic-tac-toe.py

```diff
@@ -55,6 +82,7 @@ def check_fied(avaliable_fields, ch_field):
     click = player_move()
     if click != None:
         check_fied(avaliable_fields, click)
+        draw_cross(x, y)
```

Comment:
x and y - unknown variables.
When you'll call this function, you'll get an error, cause of undefined variables.